molecule.yml

```diff
@@ -17,6 +17,12 @@ docker:
       privileged: true
       volume_mounts:
         - /sys/fs/cgroup:/sys/fs/cgroup:ro
+    - name: bionic
+      image: odyssey4me/ubuntu-molecule
```

Comment:
I'll adjust this once https://github.com/paulfantom/dockerfiles/pull/1 merges and the image is published.

Reply:
PR is merged and images are public :tada:

Reply:
Awesome, thanks!